src/content/reference/app-configuration/index.md

```diff
@@ -1,14 +1,14 @@
 ---
 title: App Configuration
 description: Documentation on the various levels of App configuration and how they get merged into a final Values object.
-date: 2020-04-08
+date: 2020-04-27
 type: subsection
 ---
 
 # App Configuration Reference
 
 Giant Swarm's [App Platform](/basics/app-platform/) allows you to easily install Apps across your entire
-fleet of clusters.
+fleet of clusters. We fully support Helm as a general tool to deploy your applications as well as for our general app catalog, which you can of course also use for your own applications by creating a new catalog.
```

Comment:
```suggestion
fleet of clusters. We fully support Helm as a general tool to deploy your applications as well as for our general app catalogue, which you can of course also use for your own applications by creating a new catalogue.
```

Reply:
Marian did a suggestion you didnt validate and it was outaded by your latest commit. 

So click on `Commit suggestion` to accept it up here.